src/status_im/contexts/settings/wallet/saved_addresses/save_address/view.cljs

```diff
@@ -35,10 +35,12 @@
 
 (defn view
   []
-  (let [{:keys [address]} (rf/sub [:wallet/saved-address])
+  (let [{:keys [edit?]} (rf/sub [:get-screen-params])
+        {:keys [address name customization-color]} (rf/sub [:wallet/saved-address])
```

Comment:
@smohamedjavid after I checked save_address screen i think it's cleaner with current changes, there are some usage of `chain-short-names` and `address-without-prefix` that just by passing full-address to this screen without `network-preferences-names` will be better IMO, wdyt?

Reply:
I agree but I was thinking of skipping the calculation of the `selected-networks` state as it's the same as the `network-preferences-names` because `chain-short-names` is generated from `selected-networks`

https://github.com/status-im/status-mobile/blob/3634edab420a1498c1c797fba3d064abfd98bc3a/src/status_im/contexts/settings/wallet/saved_addresses/save_address/view.cljs#L42-L43

https://github.com/status-im/status-mobile/blob/3634edab420a1498c1c797fba3d064abfd98bc3a/src/status_im/contexts/wallet/data_store.cljs#L144-L145

One other reason why we split address to `network-prefixes` and `address-without-prefix` is when the user enters the add new saved address flow, he/she can enter an address with network prefixes already, we need to calculate the network names and add it to the initial state. For the Edit flow, I thought of skipping this step in state initialization due to the above calculation.

This is not a trivial thing and we can live with it 👍

Reply:
@smohamedjavid could you please check it again?

Reply:
Looks good 👍 

Thanks 🙏